mix.lock

```diff
@@ -1,19 +0,0 @@
-  "distillery": {:hex, :distillery, "1.4.0", "d633cd322c8efa0428082b00b7f902daf8caa166d45f9022bbc19a896d2e1e56", [:mix], []},
-  "earmark": {:hex, :earmark, "1.2.2", "f718159d6b65068e8daeef709ccddae5f7fdc770707d82e7d126f584cd925b74", [:mix], []},
-  "elixir_make": {:hex, :elixir_make, "0.4.0", "992f38fabe705bb45821a728f20914c554b276838433349d4f2341f7a687cddf", [], [], "hexpm"},
-  "esip": {:hex, :esip, "1.0.15", "82c8b0178618c10b1ac9690841d94025c982d63f8cd6c8f8bf920cf33e301658", [], [{:fast_tls, "1.0.15", [hex: :fast_tls, repo: "hexpm", optional: false]}, {:p1_utils, "1.0.9", [hex: :p1_utils, repo: "hexpm", optional: false]}, {:stun, "1.0.14", [hex: :stun, repo: "hexpm", optional: false]}], "hexpm"},
-  "ex_doc": {:hex, :ex_doc, "0.16.1", "b4b8a23602b4ce0e9a5a960a81260d1f7b29635b9652c67e95b0c2f7ccee5e81", [:mix], [{:earmark, "~> 1.1", [hex: :earmark, optional: false]}]},
-  "ezlib": {:hex, :ezlib, "1.0.2", "22004ecf553a7d831404394d5642712e2aede90522e22bd6ccc089ca410ee098", [:rebar3], [], "hexpm"},
-  "fast_tls": {:git, "https://github.com/processone/fast_tls", "f9de643f167cfa274862d323694c1bf5b57d3be2", []},
-  "fast_xml": {:git, "https://github.com/processone/fast_xml", "4fb4f1df0da4d10ca8c86c32aa84cb2229ac0b39", []},
-  "fast_yaml": {:hex, :fast_yaml, "1.0.10", "ce5d52b77cb21968c8b73aa29b39f56a4ffd7e1e11f853d5597e7277858f155e", [:rebar3], [{:p1_utils, "1.0.9", [repo: "hexpm", hex: :p1_utils, optional: false]}], "hexpm"},
-  "goldrush": {:hex, :goldrush, "0.1.9", "f06e5d5f1277da5c413e84d5a2924174182fb108dabb39d5ec548b27424cd106", [:rebar3], [], "hexpm"},
-  "iconv": {:hex, :iconv, "1.0.5", "ae871aa11c854695db37e48fd5e5583b02e106126fbdf21bb53448f5a47c092b", [:rebar3], [{:p1_utils, "1.0.9", [repo: "hexpm", hex: :p1_utils, optional: false]}], "hexpm"},
-  "jiffy": {:hex, :jiffy, "0.14.11", "919a87d491c5a6b5e3bbc27fafedc3a0761ca0b4c405394f121f582fd4e3f0e5", [:rebar3], [], "hexpm"},
-  "lager": {:hex, :lager, "3.4.2", "150b9a17b23ae6d3265cc10dc360747621cf217b7a22b8cddf03b2909dbf7aa5", [:rebar3], [{:goldrush, "0.1.9", [repo: "hexpm", hex: :goldrush, optional: false]}], "hexpm"},
-  "p1_oauth2": {:hex, :p1_oauth2, "0.6.1", "4e021250cc198c538b097393671a41e7cebf463c248980320e038fe0316eb56b", [:rebar3], [], "hexpm"},
-  "p1_utils": {:hex, :p1_utils, "1.0.9", "c33c230efbeb4dcc02911161e3cb1a93231a92df15e3fc97de655a9271a26d9f", [:rebar3], [], "hexpm"},
-  "stringprep": {:hex, :stringprep, "1.0.9", "9182ba39931cd1db528b8883cad0d63530abe2bf21835d26cec2f9af8bc00be0", [:rebar3], [{:p1_utils, "1.0.9", [repo: "hexpm", hex: :p1_utils, optional: false]}], "hexpm"},
-  "stun": {:hex, :stun, "1.0.14", "6dc2080c25a72f7087301dc7333c1ea7d27ea4d88efaa379fc2b5924f3b17006", [], [{:fast_tls, "1.0.15", [hex: :fast_tls, repo: "hexpm", optional: false]}, {:p1_utils, "1.0.9", [hex: :p1_utils, repo: "hexpm", optional: false]}], "hexpm"},
-  "xmpp": {:git, "https://github.com/processone/xmpp", "b8debe2790f836dcb6b2234a57565b2870b21677", []}}
```

Comment:
what's going on here?

Reply:
Dependency resolution for elixir's `mix` utility.  I cleaned my directory.  Will add to the git ignore.